front/js/components/order.es6

```diff
@@ -48,7 +48,7 @@
       'onCartUpdate', renderTable, fillSavedInputs,
       touchSpinReinit, restoreSelectedPayment, cityAutocomplete,
     );
-    $(DOM.fullForm).submit(() => mediator.publish('onOrderSend', [getProductsData()]));
+    $(DOM.fullForm).submit(() => mediator.publish('onOrderSend'));
```

Comment:
why you preferred to send data with via backend, not frontend mediator?
Mediator approach seems more explicit

Reply:
@duker33 We still use mediator to publish onOrderSend event on submit. What did you mean?

Reply:
>  [getProductsData()]

@artemiy312 , i mean this data. We use it here to trigger event, but not for sending data

Reply:
@duker33 Now we have to send data on success-page, because we need a order's id for jsecommerce